pkgs/development/libraries/qt-5/5.8/qtbase/default.nix

```diff
@@ -15,7 +17,8 @@
   cups ? null, mysql ? null, postgresql ? null,
 
   # options
-  mesaSupported, mesa,
+  mesaSupported ? (!stdenv.isDarwin),
```

Comment:
afaik, mesa is supported for darwin, but the corresponding `mkspecs-libgl.path` has to be ported for darwin, too. I assume that the following defs in `mkspecs/common/mac.conf' have to be substituted by the mesa-buildInput:
```
QMAKE_LIBDIR            =

# sdk.prf will prefix the proper SDK sysroot
QMAKE_INCDIR_OPENGL     = \
    /System/Library/Frameworks/OpenGL.framework/Headers \
    /System/Library/Frameworks/AGL.framework/Headers/
```
Since this is an issue with all qt-5 releases, i would like to suggest a separate PR. What is your opinion?